src/SSW.Rewards.Application/Common/Interfaces/IApplicationDbContext.cs

```diff
@@ -19,5 +19,6 @@ public interface IApplicationDbContext
     DbSet<CompletedQuiz> CompletedQuizzes { get; set; }
     DbSet<SocialMediaPlatform> SocialMediaPlatforms { get; set; }
     DbSet<UserSocialMediaId> UserSocialMediaIds { get; set; }
+    DbSet<QuizAnswer> QuizAnswers { get; set; }
```

Comment:
You are missing the `SubmittedQuizAnswer` table

Reply:
Done